test/integrated-devnet-tests/with-docker-wrong-cli/hardhat.config.ts

```diff
@@ -8,7 +8,9 @@ module.exports = {
         integratedDevnet: {
             dockerizedVersion: process.env.STARKNET_DEVNET,
             url: "http://127.0.0.1:5050",
-            args: ["--accounts", "invalid_value"]
+            args: ["--accounts", "invalid_value"],
+            stdout: "logs/stdout.log",
+            stderr: "STDERR"
```

Comment:
Is this a change done to an old test? There were no changes to its check.sh, right? If so, this is kind of a breaking change then; could you adapt the code to log the last error if there is no stderr specified (if that's the problem), so that there are no changes required to the old tests?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/179/commits/e43e7a37b32981c32b815154da157d0b4dab4d65